pandas/tests/test_panel.py

```diff
@@ -1454,20 +1454,95 @@ def test_fillna(self):
         assert_frame_equal(filled['ItemA'],
                            panel['ItemA'].fillna(method='backfill'))
 
+        # Fill forward.
```

Comment:
The issue number is in `test_fillna_axis_0()`. The tests I added to `test_fillna()` don't address any particular issue. There just just isn't much testing for filling in panels yet.